demo/ez-navigation-hub.html

```diff
@@ -7,20 +7,55 @@
     <title>ez-navigation-hub demo</title>
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../element-matches/closest.js"></script><!-- for Edge 14 -->
+    <script src="../../fetch/fetch.js"></script><!-- for Safari 10.0 -->
 
     <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
     <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
     <link rel="import" href="../ez-navigation-hub.html">
 
     <style is="custom-style" include="demo-pages-shared-styles">
+        ez-navigation-hub {
+            position: relative;
+        }
+        ez-navigation-hub .ez-active-zone::after {
+            content: "Currently the active zone is " attr(data-zone-identifier);
```

Comment:
after fixing the CSS of the component, this should be removed and replaced by some CSS to visually highlight the active zone maybe the active link.